Makefile

```diff
@@ -14,6 +14,16 @@ run-app:
 	@echo "Starting the frontend app..."
 	@export $(shell cat .env | xargs) && npm run dev
 
+.PHONY: run-build
+run-build:
+	@echo "Building the frontend app..."
+	@export $(shell cat .env | xargs) && npm run build
+
+.PHONY: run-start
+run-start:
+	@echo "Starting the production build locally..."
+	@export $(shell cat .env | xargs) && npm run start
```

Comment:
Probably a nitpick: can reuse this somehow?

I imagine something like (ChatGPT version 😅 ):

```make
.PHONY: run-build run-start set-env

# Helper to export environment variables
set-env:
	@export $(shell cat .env | xargs)

.PHONY: run-build
run-build: set-env
	@echo "Building the frontend app..."
	npm run build

.PHONY: run-start
run-start: set-env
	@echo "Starting the production build locally..."
	npm run start
```

Reply:
Done! I have already reused some parts of the MakeFile 👌